osmcrops.csv

```diff
@@ -70,7 +70,7 @@ Maté,mate,mate_plants,orchard,
 Jojoba seed,jojoba,jojoba_plants,orchard,
 Cranberries,cranberry,,orchard,
 Grapefruit (inc. pomelos),grapefruit,grapefruit_trees,orchard,
-Tomatoes,tomatoe,,orchard,often in greenhouses
+Tomatoes,tomato,,orchard,often in greenhouses
```

Comment:
We have too commas here and two lines above...

Reply:
Yeah, this may be a new issue… maybe an "empty" crop that is being added here, no idea… :thinking: 

Feel free to open a new issue. It's not related to this PR, in any case.

Reply:
Yes, this is expected. It's an empty cell.

There is just no "OSM trees-key value" for a :tomato::
![grafik](https://user-images.githubusercontent.com/11966684/50712586-58953380-1072-11e9-8359-20a4a70d4d1d.png)